test.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Probably needs to be removed from the PR

Reply:
@pgleeson mdf.py(Condition) on doc_cleanup is now behaving the same as development. Some test cases were included particularly for Condition to test it. The reason for dependency and dependencies being present is, for example, abc_conditions.py, there is cond_term ('termination' Condition), which takes input argument as 'dependencies', which contains a list of conditions. https://github.com/ModECI/MDF/blob/doc_cleanup/examples/MDF/abc_conditions.py